src/Dashboard/StakePoolInfo.tsx

```diff
@@ -36,7 +36,7 @@ export const StakePoolInfo = ({stakePoolId}: {stakePoolId: string}) => {
           </View>
         </View>
 
-        {stakePoolInfo.homepage && (
+        {!!stakePoolInfo.homepage && (
```

Comment:
how does this change anything?

Reply:
Error: Text strings must be rendered within a component.